docs/extras/ecosystem/dependents.mdx

```diff
@@ -51,7 +51,7 @@ Dependents stats for `langchain-ai/langchain`
 |[e2b-dev/e2b](https://github.com/e2b-dev/e2b) | 5365 |
 |[mage-ai/mage-ai](https://github.com/mage-ai/mage-ai) | 5352 |
 |[wenda-LLM/wenda](https://github.com/wenda-LLM/wenda) | 5192 |
-|[LangChain-Chinese-Getting-Started-Guide](https://github.com/liaokongVFX/LangChain-Chinese-Getting-Started-Guide) | 5129 |
+|[liaokongVFX/LangChain-Chinese-Getting-Started-Guide](https://github.com/liaokongVFX/LangChain-Chinese-Getting-Started-Guide) | 5129 |
```

Comment:
Note: This part of the document was autogenerated. This change is good for now but the next time it will be automatically updated to the old name.